omod/src/main/webapp/resources/scripts/htmlForm.js

```diff
@@ -40,6 +40,24 @@
         return jq(".submitButton.confirm").is(":disabled");
     }
 
+
+    /**
+     * Change the hours, minutes and seconds dropdowns to new date with client timezone
+     * also change the encounterDate datepicker current date to new date with client timezone
+     **/
+    var changeTimeWidgetClientTimeZone = function (clientDateTime) {
+        getField('encounterDate.value').datepicker('setDate', clientDateTime)
+        jq("#encounterDate").find(".hfe-hours").val(clientDateTime.getHours()).change();
+        jq("#encounterDate").find(".hfe-minutes").val(clientDateTime.getMinutes()).change();
+        jq("#encounterDate").find(".hfe-seconds").val(clientDateTime.getSeconds()).change();
+    }
+
+    var splitDate= function (date) {
```

Comment:
What does this splitDate function do? Can you give it some documentation that includes sample input and output?

Reply:
I will add some information about that